src/middlewared/middlewared/utils/asyncio_.py

```diff
@@ -14,3 +15,13 @@
 
     futures = [func(arg) for arg in arguments]
     return await asyncio.gather(*futures)
+
+
+def call_later(delay, func, arguments):
```

Comment:
We are not using it anywhere so we probably can remove this until there is an actual need.